docs/configuration.md

```diff
@@ -87,6 +87,25 @@ Release-it has [plenty of options][2]. See the following tables for plugin confi
 - [GitHub](./github-releases.md#configuration-options)
 - [GitLab](./gitlab-releases.md#configuration-options)
 
+### Extend Configuration
+
+You can extend a configuration from a remote source using the `extends` option. The following formats are supported:
+
+- `github>owner/repo`
+- `github>owner/repo#tag`
+- `github>owner/repo:file#tag`
```

Comment:
How do you feel about using `:` as more of a "protocol delimiter"?

Can't remember having seen `>` myself before, and `:` is more common?

By the way we could also use something like 

- `npm:juancarlosjr97/release-it-configuration/release-it.json` and/or even
- `https://unpkg.com/juancarlosjr97/release-it-configuration/schema/release-it.json` 

for anything, also anything published on npm is accessible e.g. https://unpkg.com/release-it@17.10.0/config/release-it.json.

Reply:
I am not sure if I have seen a format like this anywhere else besides [Renovate Preset File Naming](https://docs.renovatebot.com/config-presets/#preset-file-naming). It follows the same format I proposed, which is why I went with it. That said, I am happy to change it, as it is ultimately just an abstract representation.

Having written that, I think `:` has a stronger logical association with URLs compared to `>` now that you mention it, so I will change it :) 

> By the way, we could also use something like...

I really like the idea of using `unpkg`. That way, we are not constrained to GitHub (only to its other owned package registry 😄) 

Would you like me to work on adding npm (unpkg) in the current PR, or should I raise another issue and tackle it as a separate PR? I am happy to do it either way!

Reply:
Happy to leave that up to you. Maybe good to keep this one small, but if another PR's overhead you can add it here too.